src/uct/api/uct.h

```diff
@@ -665,13 +672,22 @@ void uct_iface_mem_free(const uct_allocated_memory_t *mem);
 
 /* @ingroup UCT_AM
  * @brief  List of capabilities of active message callback
+ *
+ * A callback must have either SYNC or ASYNC flags.
```

Comment:
If there has to be a SYNC or ASYNC flag, should there be a flag at 0 called UCT_AM_CB_FLAG_INVALID?